modules/doc/content/getting_started/installation/wsl.md

```diff
@@ -1,45 +1,82 @@
 
 ## Launch VcXsrv
 
-Each time you reboot, or basically each time VcXsrv is *not* running, and you wish to use the graphical capabilities of MOOSE (Peacock), you should start VcXsrv before launching your WSL terminal.
+Each time you reboot, or basically each time VcXsrv is *not* running, and you wish to use the graphical
+capabilities of MOOSE (Peacock), you should start VcXsrv before launching your WSL terminal.
 
-We have found better performance instructing VcXsrv to use software rendering over native OpenGL. When launching VcXsrv, search for the option: "Native opengl", and un-check it. All other options can remain the default.
+When starting VcXsrv, options for the server can be adjusted. In general, the default options are
+adequate, except for the following:
 
-## Edit Hostname within WSL
+- Check "Disable access control". This allows the WSL instance to open windows via VcXsrv without
+  requiring an authorization protocol.
 
-Launch the Windows 10 Command Prompt as an Administrator (right-click the Windows Start button, and select Command Prompt(Admin) in the resulting menu that appears), and modify the Windows 10 hosts file (located in `C:\Windows\System32\Drivers\etc\hosts`) to include the results of `hostname` to resolve to 127.0.0.1. This is necessary due to the way
-MPICH (a message passing interface) passes information among itself when running applications (like MOOSE) in parallel.
+Again, all other options for VcXsrv can remain the default.
 
+!alert tip title=VcXsrv network connections
+In some cases, the system firewall may block connections to the X window server. If errors or window
+rendering stalls are experienced after adopting the VcXsrv settings above, check to make sure that all
+network connections to VcXsrv are allowed.
+
+## Install WSL
+
+!alert note
+To use the simplified installation instructions outlined below, a recent build of Windows should be
+used (10 version 2004+ and build 19041+, or 11). For more information see
+[Microsoft's information page on WSL](https://learn.microsoft.com/en-us/windows/wsl/install).
+
+Open PowerShell or Windows Command Prompt in *administrator* mode by right-clicking the application
+and selecting "Run as administrator". In the prompt that appears, one command can be used to install
+WSL dependencies and a default Linux distribution of Ubuntu 20.04 LTS.
+
+```bash
+wsl --install
 ```
-C:\Users\[your_user_name]> hostname
-DESKTOP-L7BGA7L
 
-C:\Users\[your_user_name]> Notepad "C:\Windows\System32\Drivers\etc\hosts"
-# localhost name resolution is handled within DNS itself.
-#       127.0.0.1       localhost
-#       ::1             localhost
+Program output will appear showing installation and setup of dependencies before prompting the user
+for a restart to finish initial setup. See below for an example of the summary output.
 
-127.0.0.1   DESKTOP-L7BGA7L    <---- ADD THAT
+```bash
+C:\Users\USER>wsl --install
```

Comment:
The instructions I followed from Microsoft also asked me to run these commands:
```bash
PS C:\Windows\system32> wsl --update
PS C:\Windows\system32> wsl --set-default-version 2
PS C:\Windows\system32> wsl --shutdown
```
Maybe we should add them?

Reply:
It's funny, because I didn't see those in the instructions I found from Microsoft. These just confirm that you want to set the WSL version to be 2, which it is now by default. I don't mind adding them, since it shouldn't make a difference, but it does give us a line to point to if someone wants to go back to WSL1 for whatever reason.

Reply:
Also you can run commands in either powershell or Command Prompt now (where it was only PS before). Pretty neat changes that make it way more useful!

Reply:
See f7f3726. I added these commands with individual context at the bottom of the page as a set of "miscellaneous tips". Let me know what you think!